slices/fonts-urw-base35.yaml

```diff
@@ -0,0 +1,21 @@
+package: fonts-urw-base35
+
+essential:
+  - fonts-urw-base35_copyright
+
+slices:
+  config:
+    contents:
+      /etc/fonts/conf.d/61-urw*:
+      /usr/share/fontconfig/conf.avail/urw-*:
+
+  fonts:
+    contents:
+      /etc/X11/fonts/Type1/fonts-urw-base35*:
```

Comment:
Shouldn't this be in `config`?

Reply:
Had been renamed by request (so data split to fonts and config).